recipe/meta.yaml

```diff
@@ -32,7 +32,7 @@ requirements:
     - pywin32-ctypes  # [win]
     - pip
     - cython >=0.21
-    - numpy 1.11.*
+    - numpy 1.14.*
```

Comment:
Just unpin this and re-render. Conda-forge will handle that for you.

Reply:
Thanks @ocefpaf !